numpy/core/tests/test_multiarray.py

```diff
@@ -23,6 +23,14 @@
 
 from numpy.compat import pickle
 
+try:
+    import pathlib
+except ImportError:
+    try:
+        import pathlib2 as pathlib
```

Comment:
I'm not actually sure our stuff works on pathlib2, or whether our CI exercises this code path.

Either way, if it turns out this test fails on systems using pathlib2, at least with the test present we'll likely get a bug report.